vendors/cypress/boards/CYW943907AEVAL1F/ports/wifi/iot_wifi.c

```diff
@@ -128,13 +161,14 @@ WIFIReturnCode_t WIFI_ConnectAP( const WIFINetworkParams_t * const pxNetworkPara
     wiced_result_t      result;
     wiced_scan_result_t ap_info;
     wiced_ap_info_t     ap;
+    char pcSSID[ pxNetworkParams->ucSSIDLength + 1 ];
```

Comment:
this could cause stack overflow if a large SSID length is passed to the API. Could we declare array static with configured max wifi ssid length.

Reply:
Good catch. These were intended to be guarded to bound size but some, such as this board, were missing that